src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerSelector.iOS.cs

```diff
@@ -42,7 +42,8 @@ private protected override void OnLoaded()
 			SetPickerTime(Time.RoundToNextMinuteInterval(MinuteIncrement));
 			SaveInitialTime();
 			_picker.ValueChanged += OnValueChanged;
-			
+			_picker.EditingDidBegin += OnEditingDidBegin;
```

Comment:
Should we avoid registering if `UseLegacyStyle` is `true` ?

Reply:
Makes no difference. The Legacy Picker is a Wheel that does not trigger the keyboard